### PR TITLE
feat(ui): Add Bahasa Indonesia localization language support

### PR DESCRIPTION
**Pull request by ekasetiawans**
## Description

*Replace this paragraph with a description of what this PR is doing. If you're modifying existing behavior, describe the existing behavior, how this PR is changing it, and what motivated the change.*

## Related Issues

*Replace this paragraph with a list of issues related to this PR from the [issue database](https://github.com/flutter/flutter/issues). Indicate, which of these issues are resolved or fixed by this PR. Note that you'll have to prefix the issue numbers with flutter/flutter#.*

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`).
This will ensure a smooth and quick review process. Updating the `pubspec.yaml` and changelogs is not required.

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [ ] My PR includes unit or integration tests for *all* changed/updated/fixed behaviors (See [Contributor Guide]).
- [ ] All existing and new tests are passing.
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [ ] The analyzer (`melos run analyze`) does not report any problems on my PR.
- [ ] I read and followed the [Flutter Style Guide].
- [ ] I signed the [CLA].
- [ ] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Does your PR require plugin users to manually update their apps to accommodate your change?

- [ ] Yes, this is a breaking change.
- [x] No, this is *not* a breaking change.

[issue database]: https://github.com/flutter/flutter/issues
[Contributor Guide]: https://github.com/FirebaseExtended/flutterfire/blob/master/CONTRIBUTING.md
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[pub versioning philosophy]: https://dart.dev/tools/pub/versioning
[CLA]: https://cla.developers.google.com/

### CODE REVIEW

File: packages/flutterfire_ui/lib/src/i10n/default_localizations.dart [hunk 1]
```diff
@@ -1,3 +1,5 @@
+import 'package:flutterfire_ui/src/i10n/lang/id.dart';
```
Comment (lesnitsky) [inline packages/flutterfire_ui/lib/src/i10n/default_localizations.dart:1]:
```suggestion
import 'lang/id.dart';
```

File: packages/flutterfire_ui/lib/src/i10n/lang/id.dart [hunk 2]
```diff
@@ -0,0 +1,263 @@
...
+    this.passwordResetEmailSentText =
+        'Kami telah mengirim tautan untuk mengatur ulang kata sandi ke surel Anda. Mohon periksa surel Anda.',
+    this.forgotPasswordHintText =
+        'Masukkan surel Anda, dan kami akan mengirimkan tautan untuk mengatur ulang kata sandi ke email Anda',
+    this.emailLinkSignInButtonLabel = 'Masuk dengan magic link',
+    this.signInWithEmailLinkViewTitleText = 'Masuk dengan magic link',
+    this.signInWithEmailLinkSentText =
+        'Kami telah mengirim magic link ke surel Anda. Periksa email Anda dan ikuti tautan untuk masuk',
+    this.sendLinkButtonLabel = 'Kirim magic link',
+    this.arrayLabel = 'array',
```
Comment (lesnitsky) [inline packages/flutterfire_ui/lib/src/i10n/lang/id.dart:243]: @ekasetiawans would you be able to translate this as well as the labels below?

**Review by lesnitsky — CHANGES_REQUESTED** [review]
DataTable labels require translations

**Comment by ekasetiawans** [comment]
@lesnitsky i did it

**Review by ekasetiawans — COMMENTED** [review]
thanks

**Review by ekasetiawans — COMMENTED** [review]
okay

**Comment by russellwheatley** [comment]
Hey @ekasetiawans, thanks for the PR! I don't suppose you are able to have your Bahasa Indonesia double checked by someone who speaks the language (i.e get this PR approved by a developer who speaks the language)? Just for spellchecking and correct language interpretation. Sadly, I don't know anyone from Indonesia 😄

**Comment by ekasetiawans** [comment]
@russellwheatley do you mean anyone from Indonesia or just for collaboratos from Indonesia?

**Comment by russellwheatley** [comment]
@ekasetiawans, someone you know would be fine. I prefer a proofreading whenever there's a labelling/documentation PR.

**Comment by ekasetiawans** [comment]
@russellwheatley I'm very new to open source collaboration, so I need some guidance in case something goes wrong.

**Comment by RTheTopanX404** [comment]
@ekasetiawans  ..
I'm already check all of translations to Indonesia, and everything is translated to Indonesia .

**Pull request merged**